index.html

```diff
@@ -5,7 +5,7 @@
         <!-- begin banner announcement -->
         <a href="#" target="_blank">
             <section class="announcement">
-                <span><i class="fa fa-bullhorn"></i> IMPORTANT: This is an important announcement, yo. <span class="styled-link">Here is the link providing more details that ya’ll should read</span></span>
+                <span><i class="fa fa-bullhorn"></i> IMPORTANT: End-of-life for all Python 2-based versions of OMERO <span class="styled-link"><a href="{{ site.baseurl }}/2019/11/13/python-3-rollout.html">Read the Announcement</a> </span></span>
```

Comment:
@lunson: this is currently looking pale

<img width="571" alt="Screen Shot 2019-11-13 at 21 59 07" src="https://user-images.githubusercontent.com/88113/68803886-e3669780-0660-11ea-94bc-d877b59ceef5.png">

Reply:
Gah! Looked fine on my local, sorry about that. This should be fixed now - white for maximum contrast.